lua/gitblame/init.lua

```diff
@@ -114,7 +115,7 @@ local function process_blame_output(blames, filepath, lines)
     end
 
     if not files_data[filepath] then
-        files_data[filepath] = {}
+        files_data[filepath] = { blames = {} }
```

Comment:
Luals still assumes it is an error since `blames` cannot be `nil` for `GitInfo`, so it needs to be at least declared to some initial value here. I can also make `blames` nullable and remove this, although I think it's nice to always have a guarantee that it's not null. Up to you!

Reply:
Gotcha! The current implementation is better, then!